persistence/build.gradle

```diff
@@ -1,3 +1,13 @@
 dependencies {
     compile project(':core-models')
-}
\ No newline at end of file
+    implementation project(':metrics')
+    implementation project(':test-utils')
+
+    compile "org.scala-lang.modules:scala-async_$scalaVersion:0.9.7"
+    compile "com.wix:accord-core_$scalaVersion:0.7.1"
+
+    testCompile "com.typesafe.akka:akka-testkit_$scalaVersion:2.5.14"
```

Comment:
testing libs should be defined for the whole project since multiple modules will use them.

Reply:
Yeah, I'm thinking this dep should be in test-utils

Reply:
Done